### PR TITLE
FIX Handle admin_email array config

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10257

### CODE REVIEW

File: src/Control/Email/Email.php [hunk 1]
```diff
@@ -289,8 +289,19 @@ public function setSwiftMessage($swiftMessage)
      */
     private function getDefaultFrom(): string
     {
-        $defaultFrom = $this->config()->get('admin_email');
-        if (!$defaultFrom) {
+        // admin_email can have a string or an array config
+        // https://docs.silverstripe.org/en/4/developer_guides/email/#administrator-emails
+        $adminEmail = $this->config()->get('admin_email');
+        if (is_array($adminEmail) && count($adminEmail) > 0) {
+            $defaultFrom = array_keys($adminEmail)[0];
```
Comment (michalkleiner) [inline src/Control/Email/Email.php:296]:
```suggestion
            $defaultFrom = array_key_first($adminEmail);
```

**Review by michalkleiner — APPROVED** [review]

File: src/Control/Email/Email.php (hunk 1 above)
Comment (emteknetnz) [reply src/Control/Email/Email.php]: Updated

Comment (emteknetnz) [reply src/Control/Email/Email.php:296]: Reverted, PHP 7.1 (which 4.9 supports) doesn't have the array_key_first, it's new with php 7.3.0

Comment (michalkleiner) [reply src/Control/Email/Email.php:296]: Ah, ok, I thought we dropped support below 7.4 now but didn't realise 4.9 still has it. All good.

Comment (GuySartorelli) [reply src/Control/Email/Email.php:296]: Yeah 4.10 was when older PHP support was dropped.

**Pull request merged**